src/app/views.py

```diff
@@ -191,10 +191,9 @@ def submitNewLicense(request):
             """ Other errors raised """
             logger.error(str(format_exc()))
             if (request.is_ajax()):
-                ajaxdict["type"] = "error"
-                ajaxdict["data"] = "Unexpected error, please email the SPDX technical workgroup that the following error has occurred: " + format_exc()
-                response = dumps(ajaxdict)
-                return HttpResponse(response,status=500)
+                ajaxdict['error'] = str(format_exc())
+                ajaxdict['inputlicensetext'] = request.POST['text']
+                return JsonResponse(ajaxdict)
```

Comment:
Does the caller expects "type" and "data" field now?

Reply:
@Ugtan What about this?

Reply:
@rtgdk No. I had to use Json response to get the data after making a ajax request in order to give them the ability to choose if they want to submit it or not.